src/asmcnc/apps/drywall_cutter_app/widget_drywall_shape_display.py

```diff
@@ -481,21 +485,25 @@ def select_shape(self, shape, rotation, swap_lengths=False):
             if rotation == 'horizontal':
                 self.enable_input(self.l_input, (240, 228))
                 self.place_widget(self.x_datum_label, (414, 75))
-                self.place_widget(self.y_datum_label, (425, 195))
+                self.place_widget(self.y_datum_label, (422, 195))
+                self.place_widget(self.y_datum_validation_label, (422, 195 - 35))
```

Comment:
(self.y_datum_label.x, self.y_datum_label.y - 35) and possibly replace 35 with a variable